tests/parser/functions/test_create_functions.py

```diff
@@ -330,7 +329,7 @@ def should_fail(target: address, arg1: String[129], arg2: Bar):
     assert test.foo() == FOO
     assert test.bar() == BAR
 
-    encoded_args = encode_single("(string,(string))", (FOO, BAR))
+    encoded_args = abi_encode("(string,(string))", [FOO, BAR])
```

Comment:
this fix doesn't seem right. should it be this?
```suggestion
    encoded_args = abi_encode("(string,(string))", (FOO, (BAR,)))
```

Reply:
I think this is correct since `BAR` is already defined as a tuple above: `BAR = ("world!",)`

Reply:
i mean, `(FOO, BAR)` instead of `[FOO, BAR]`